Doc/library/sqlite3.rst

```diff
@@ -1003,20 +1004,19 @@ Cursor Objects
 
    .. method:: fetchone()
 
-      Fetches the next row of a query result set, returning a single sequence,
-      or :const:`None` when no more data is available.
+      Fetch the next row of a query result set as a :class:`tuple`.
```

Comment:
We _could_ also mention that the returned row may be produced by a custom `row_factory`.

Reply:
Ditto for fetchmany and fetchall.